extensions/amp-analytics/0.1/vendors.js

```diff
@@ -454,6 +454,59 @@ export const ANALYTICS_CONFIG = {
     },
   },
 
+  'mparticle': {
+    'vars': {
+      'apiKey': '',
+      'eventName': '',
+      'eventType': 'Unknown',
+      'eventAttributes_Keys': [],
```

Comment:
Empty strings and arrays have the same effect as not specifying them at all. Remove them from here?